Makefile

```diff
@@ -0,0 +1,30 @@
+CONTAINER_ENGINE?=podman
+EXECUTABLE:=log-exploration-oc-plugin
+PACKAGE:=github.com/ViaQ/log-exploration-oc-plugin
+IMAGE_PUSH_REGISTRY:=quay.io/emishra/$(EXECUTABLE)
+VERSION:=${shell git describe --tags --always}
+BUILDTIME := ${shell date -u '+%Y-%m-%d_%H:%M:%S'}
+BUILD_DIR:=./bin
```

Comment:
`BUILD_DIR` is not used. I guess you wanted to add it to the `build` target? Like `go build -o $(BUILD_DIR)/$(EXECUTABLE) cmd/oc-historical_logs.go`

Reply:
Thanks for the review @igor-karpukhin. I'll update this.